pkgs/development/python-modules/pyopengltk/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pyopengl,
+  writers,
+  tkinter,
+  pyopengltk,
+}:
+
+buildPythonPackage rec {
+  pname = "pyopengltk";
+  version = "0.0.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jonwright";
+    repo = "pyopengltk";
+    rev = "dbed7b7d01cc5a90fd3e79769259b1dc0894b673"; # there is no tag
+    hash = "sha256-hQoTj8h/L5VZgmq7qgRImLBKZMecrilyir5Ar6ne4S0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pyopengl
+    tkinter
+  ];
+
```

Comment:
Could you enable `pythonImportsCheck`?

You can see a checklist in the contribution guide.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
I keep forgetting that one everytime